### ISSUE TITLE
Prisma generate throws error when only ID is provided in the datamodel

### ISSUE THREAD

**Issue by pantharshit00** [prisma/prisma#3872]
**Describe the bug**
`prisma generate` will throw an error if there are no other fields than ID in the datamodel.

```graphql
type Test{
  id: ID! @unique
}
```

Generating a client from this datamodel will result in an error:
```sh
 TypeError: Cannot read property 'type' of undefined
        at TypescriptGenerator.renderFieldType (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:607:13)
        at TypescriptGenerator.renderInputFieldTypeHelper (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:430:17)
        at TypescriptGenerator.renderArgs (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:402:53)
        at C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:454:44
        at Array.map (<anonymous>)
        at TypescriptGenerator.renderMainMethodFields (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:452:8)
        at TypescriptGenerator.renderMutations (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:312:17)
        at TypescriptGenerator.render (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-client-lib\src\codegen\typescript-client.ts:179:28)
    at GenereateCommand.<anonymous> (C:\Users\Scott\AppData\Roaming\npm\node_modules\prisma\node_modules\prisma-cli-core\src\commands\generate\generate.ts:181:28)
```

It is mainly caused by https://github.com/prisma/prisma/blob/9756edd60057e16804f649c01b5564660b3a415e/cli/packages/prisma-client-lib/src/codegen/typescript-client.ts#L607. There should be check for nullability there.

**To Reproduce**
Just try to generate a client from above datamodel

**Expected behavior**
Since Prisma server thinks this datamodel is valid the generate command should not throw any error.

**Versions (please complete the following information):**
 - Connector: all
 - Prisma Server: 1.24.0
 - `prisma` CLI: 1.24.0

**Additional context**
https://www.prisma.io/forum/t/prisma-generate-not-working/5678

**Comment by chrischen** [comment]
I would like to add it seems to not just be id field but also any special fields such as createdAt, or updatedAt, while at the same time an absence of user-defined fields.

**Comment by divyenduz** [comment]
This is resolved in the latest CLI beta `1.26.0-beta.1`, please install it via `npm install -g prisma`. Please let me know if that works for you.

**Comment by divyenduz** [comment]
This is fixed in the latest stable, `1.26.x`. Please install it via `npm install -g prisma`.
